.travis.yml

```
@@ -20,6 +20,11 @@ before_install:
     wget https://repo.continuum.io/miniconda/Miniconda3-latest-${arch}-x86_64.sh -O miniconda.sh
     bash miniconda.sh -b -p ~/mc
     source ~/mc/etc/profile.d/conda.sh
+  - |
+    echo "machine github.com" > $HOME/.netrc
+    echo "login $GITHUB_USERNAME" >> $HOME/.netrc
+    echo "password $GITHUB_TOKEN" >> $HOME/.netrc
+    chmod og-rw $HOME/.netrc
```

This will verbously report the changes/retained permissions. Also, better to follow the order "user" - "group" - "others" permission bits. Alternatively, can be `chmod -v 600 $HOME/.netrc`.
```suggestion
    chmod -v go-rw $HOME/.netrc
```